src/core/mount.c

```diff
@@ -1997,6 +1997,7 @@ static void mount_enumerate(Manager *m) {
         mnt_init_debug(0);
 
         if (!m->mount_monitor) {
+                uint64_t mount_rate_limit_interval = 1 * USEC_PER_SEC;
```

Comment:
actually use `usec_t` for this type. it's the internal type we use for all time values. it's actually the same as `uint64_t`, but it's more descriptive internally.

Yes, sd_event_source_set_ratelimit() actually does take `uint64_t` rather than `usec_t`, but that's just because that's a public-facing API, and `usec_t` is our internal type. But given this here is internal code, use `usec_t` please
(